keyboards/candybar/readme.md

```diff
@@ -15,4 +15,4 @@ Make example for this keyboard (after setting up your build environment):
 
     make candybar:default:dfu-util
 
-See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
+See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/newbs_building_firmware) for more information.
```

Comment:
We should align this to the [current template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md)
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```